### ISSUE TITLE
Fix Inspector section @ this breakpoint

### ISSUE THREAD

**Issue by jdaudier** [marionettejs/marionettejs.com#266]
This broke for some reason... 
![2015-02-18_1402](https://cloud.githubusercontent.com/assets/1666031/6254492/0966cd08-b777-11e4-884f-0a339661c58d.png)

Used to be like this
![old](https://cloud.githubusercontent.com/assets/1666031/6254651/6199207e-b778-11e4-87cd-e31e20755e28.png)

**Comment by peterblazejewicz** [comment]
@jdaudier Can you test local version (from master)? It appears to be ok:
![image](https://cloud.githubusercontent.com/assets/14539/6255715/9a13dd60-b7b2-11e4-9ab6-5c969b8dfae7.png)

**Comment by jdaudier** [comment]
@peterblazejewicz yup, worked locally.  The _ui.scss styles needs to go first (which it does in the local version) but in the deployed site the UI styles went later.

**Comment by peterblazejewicz** [comment]
@jdaudier 
That's also due to extracted css by `critical` task - but this time there is no tool fault in this.
We should refactor our code - note: the css contains information about positioning/spacing button directly on its instance - instead of applying it to a container (column or list) - so it gets messed:
![20150221205857](https://cloud.githubusercontent.com/assets/14539/6315754/9aad9674-ba0c-11e4-9f74-094a938af808.jpg)
we should avoid to put spacing (where UI element should be) directly on element itself - we will have to introduce containers (they are more flexible to adapt to responsive layouts I think)

**Comment by jdaudier** [comment]
@peterblazejewicz are you saying we need to wrap the action buttons inside another div and move the margin CSS to affect that div?

**Comment by peterblazejewicz** [comment]
yes, something like this. That could allow us to avoid a problems with specificity order - see screenshot - the critical css and later injected main CSS are rendered in different order than we expect. Looking into this right now.

![image](https://cloud.githubusercontent.com/assets/14539/6315912/170199fa-ba12-11e4-98c5-1387a31a5a78.png)

**Comment by peterblazejewicz** [comment]
@samccone I think we need to update `critical` task options - it looks like we should take care of duplicate rules/declarations and possible conflicts like here. I've read this comment in `loadCSS` source:
https://github.com/filamentgroup/loadCSS/blob/master/loadCSS.js#L8-L14
which exactly details situation we have I think.

**Comment by jdaudier** [comment]
:confused:

**Comment by peterblazejewicz** [comment]
@jdaudier Here is an example how we could avoid problems:
https://github.com/peterblazejewicz/marionettejs.com/tree/refactor/navigation-wip
- the buttons are abstracted into UI components
- there is abstract navigation element that works like containers to hold UI elements (this one could be a div with similar purpose like row+cols)
(only header/masthead section is refactored btw, that's just sample)

**Comment by jdaudier** [comment]
gotcha. thanks for the example! 🀌 (woah, mahjong emoji)

**Comment by peterblazejewicz** [comment]
@jdaudier hints for headers to avoid the same thing:
https://www.youtube.com/watch?v=SqfhZk0eIdE&t=366
